scripts/js/interfaces.js

```diff
@@ -46,21 +47,32 @@ $(function () {
 
       const status = `<span class="${carrierColor}">${stateText}</span>`;
 
+      let master = null;
+      if (interface.master !== undefined) {
+        // Find interface.master in data.interfaces
+        master = data.interfaces.find(obj => obj.index === interface.master).name;
+      }
+
+      // Show an icon for indenting slave interfaces
+      const indentIcon = master === null ? "" : "<i class='fa fa-diagram-project fa-fw'></i> ";
```

Comment:
Suggestion to use indentation + network icon:

![image](https://github.com/user-attachments/assets/0ab44914-4beb-40d1-8dbc-169a2d5ec7be)


1. use `fa-network-wired` as secondary icon for child interfaces:
```suggestion
      const indentIcon = master === null ? "" : "<i class='fa fa-network-wired fa-fw'></i> ";
```